cmd/contour/servecontext.go

```diff
@@ -320,7 +321,9 @@ func (ctx *serveContext) grpcOptions() []grpc.ServerOption {
 // for tls communication, tlsconfig returns nil.
 func (ctx *serveContext) tlsconfig() *tls.Config {
```

Comment:
I think maybe changing `tlsconfig` to take a `log logrus.FieldLogger` param is better here.

Reply:
done